entity-service/src/test/kotlin/com/egm/stellio/entity/web/EntityHandlerTests.kt

```diff
@@ -346,6 +349,77 @@ class EntityHandlerTests {
             )
     }
 
+    @Test
+    fun `get entity by id should correctly serialize multi-attribute property having one instance`() {
+        every { entityService.exists(any()) } returns true
+        every { entityService.getFullEntityById(any()) } returns ExpandedEntity(
+                mapOf(
+                        "https://uri.etsi.org/ngsi-ld/name" to
+                                mapOf(
+                                        NGSILD_ENTITY_TYPE to "https://uri.etsi.org/ngsi-ld/Property",
+                                        NGSILD_PROPERTY_VALUE to "ruche",
+                                        NGSILD_DATASET_ID_PROPERTY to mapOf(
+                                            NGSILD_ENTITY_ID to "urn:ngsi-ld:Property:french-name"
+                                        )
+                                ),
+                        NGSILD_ENTITY_ID to "urn:ngsi-ld:Beehive:4567",
+                        NGSILD_ENTITY_TYPE to listOf("Beehive")
+                ),
+                listOf(NGSILD_CORE_CONTEXT)
+        )
+
+        webClient.get()
+                .uri("/ngsi-ld/v1/entities/urn:ngsi-ld:BeeHive:TESTC")
+                .header(HttpHeaders.CONTENT_TYPE, MediaType.APPLICATION_JSON_VALUE)
+                .exchange()
+                .expectStatus().isOk
+                .expectBody().json(

```

Comment:
I meant using a Kotlin multi-line string, not to add carriage returns (which is still bloated)!